coral/src/app/features/requests/components/DeleteRequestModal.tsx

```diff
@@ -0,0 +1,27 @@
+import { Modal } from "src/app/components/Modal";
```

Comment:
I think we agreed on using a `Dialog?` with `type={"danger"}` :o https://github.com/aiven/klaw/issues/767#issuecomment-1458233014

Reply:
🤦‍♀️ 🤦‍♀️ 🤦‍♀️ 🤦‍♀️  now that I see the post, I remember we aligned on that, yes! 
I even started with the Dialog -> but then I looked into  Figma and changed it back again. 

We really need one source of truth with designs. 

Will update my PR.